rt_dashboard/templates/rt_dashboard/dashboard.js

```diff
@@ -257,7 +274,14 @@ var api = {
         e.stopPropagation();
 
         var $this = $(this);
-        $.post($this.attr('href'), function(data) {
+        const request = new Request(
+            $this.attr('href'),
+            {headers: {'X-CSRFToken': csrftoken}}
+        );
+        fetch(request, {
```

Comment:
do we need to support IE11?

probably not since it will be dead in ~7~8months
https://blogs.windows.com/windowsexperience/2021/05/19/the-future-of-internet-explorer-on-windows-10-is-in-microsoft-edge/